src/oled.c

```diff
@@ -12,11 +12,19 @@ uint8_t oled_init_array[] = {
     0x80, 0x52, /*Divide ratios*/
     0x80, 0xA8, /*Set Multiplex Ratio*/
     0x80, 0x0F, /*16 == max brightness,39==dimmest*/
+#ifdef OLED_FLIP
+    0x80, 0xC8, /*Set COM Scan direction backwards*/
```

Comment:
It seems to have no effect changing this value, since the output is always the same.
I tried:
**`0xC1`**, **`C2`**, **`C3`**, **`C8`**, **`C9`** & **`D0`**